docs/source/conf.py

```diff
@@ -2097,6 +2097,7 @@
     "NoneType",
     "NoopLogger",
     "NumberType",
+    "NumpyEncoder",
```

Comment:
Please do not add anything into this file.
You should either make this private (add an _ at the beginning) or properly document it.